lib/src/sdk_methods.dart

```diff
@@ -0,0 +1,16 @@
+enum SdkMethod {
+  initialize,
+  getAvailability,
+  getDeviceId,
+  start,
+  stop,
+  setAvailability,
+  setDeviceName,
+  isRunning,
```

Comment:
should we have the isRunning? I thought we don't want to confuse users and only expose isTracking cc @tulushev (maybe i'm misremembering tho)

Reply:
to be added

Reply:
isRunning == true when isTracking == false && isAvailable == true